build_and_push.sh

```diff
@@ -6,8 +6,29 @@ set -o pipefail
 IMG="lidofinance/oracle:latest"
 export DOCKER_CONFIG=$HOME/.lidofinance
 
+commit_short_hash=$(git log -1 --format=%h)
+commit_datetime=$(git show -s --format=%ci $commit_short_hash)
+#commit_date=$(echo $commit_datetime | awk '{print $1}')
```

Comment:
Since it's valuable just for you only, commented code should be separated (either unstaged or stashed) before commiting.